<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventana Principal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>471</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 101, 164);
background-color: rgb(85, 85, 0);
color: rgb(255, 255, 255);
border-radius: 25%;
font: 65 italic 20pt &quot;URW Bookman L&quot;;
border-style: solid;
border-width: 2px;</string>
    </property>
    <property name="text">
     <string>Ingresar Pelicula</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>471</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 101, 164);
background-color: rgb(85, 85, 0);
color: rgb(255, 255, 255);
border-radius: 25%;
font: 65 italic 20pt &quot;URW Bookman L&quot;;
border-style: solid;
border-width: 2px;
</string>
    </property>
    <property name="text">
     <string>Ver o Editar el Catologo de Pelicula</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProd">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>471</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 101, 164);
background-color: rgb(85, 85, 0);
color: rgb(255, 255, 255);
border-radius: 25%;
font: 65 italic 20pt &quot;URW Bookman L&quot;;
border-style: solid;
border-width: 2px;</string>
    </property>
    <property name="text">
     <string>Arboles de Todas Las Peliculas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAbout">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 101, 164);
background-color: rgb(85, 85, 0);
color: rgb(255, 255, 255);
border-radius: 25%;
font: 65 italic 20pt &quot;URW Bookman L&quot;;
border-style: solid;
border-width: 2px;</string>
    </property>
    <property name="text">
     <string>Creado por</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ForbiddenCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 11pt &quot;URW Palladio L&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Cant. Peliculas :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="NumProLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>400</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ForbiddenCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 49pt &quot;URW Bookman L&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAtras">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 0, 0);
border-radius: 25%;
font: 65 italic 20pt &quot;URW Bookman L&quot;;
border-style: solid;
border-width: 2px;</string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
